docs/configuration/OVERVIEW.md

```diff
@@ -7,25 +7,25 @@ application/components.
 ## Component config types
```

Comment:
Remark for this:
![image](https://github.com/AmadeusITGroup/otter/assets/110408057/754d7d90-e3d2-4384-8a05-66747ccd5b15)
There is a missing article or plural